src/anomalib/metrics/base.py

```diff
@@ -109,6 +132,8 @@ def __init__(
             raise ValueError(msg)
         self.fields = fields
         self.name = prefix + self.__class__.__name__
+        self.strict = strict
+        self.__update_count = 0  # keeps track of the update calls of the wrapper class
```

Comment:
is double underscore `__` intentional?

Reply:
yes, but it's no longer needed, so I'll remove it